vpr/src/route/route_common.cpp

```diff
@@ -713,24 +690,49 @@ float get_rr_cong_cost(int inode) {
                     continue; //Already included above
                 }
 
-                cost += get_single_rr_cong_cost(node);
+                cost += get_single_rr_cong_cost(node, pres_fac);
             }
         }
     }
     return (cost);
 }
 
-/* Returns the congestion cost of using this rr_node, *ignoring*
- * non-configurable edges */
-static float get_single_rr_cong_cost(int inode) {
+/* Returns the congestion cost of using this rr_node,
+ * *ignoring* non-configurable edges */
+float get_single_rr_cong_cost(int inode, float pres_fac) {
```

Comment:
Not necessary at this point, but you may want to explore putting this function in a header file with an inline in front of it to see if it affects speed (if you're calling it across file boundaries).
With interprocedural optimization on (the default in our makefile) it may not make any difference though.